collectors/node.go

```diff
@@ -266,28 +266,3 @@ func (nc *nodeCollector) collectNode(ch chan<- prometheus.Metric, n v1.Node) {
 	addResource(descNodeStatusAllocatableMemory, n.Status.Allocatable, v1.ResourceMemory)
 	addResource(descNodeStatusAllocatablePods, n.Status.Allocatable, v1.ResourcePods)
 }
-
-// addConditionMetrics generates one metric for each possible node condition
-// status. For this function to work properly, the last label in the metric
-// description must be the condition.
-func addConditionMetrics(ch chan<- prometheus.Metric, desc *prometheus.Desc, cs v1.ConditionStatus, lv ...string) {
```

Comment:
did you mean to put this into `collectors.go`?

Reply:
Ah moved it into it's own file and forget to add it. Fixed! Or would you rather have it in collectors.go?

Reply:
Either works for me - no strong opinion.

Reply:
Moving to a separate file will probably break e2e tests due to changes I have made recently. But it should be easy to fix that.

Reply:
I've just moved it to collectors.go